src/view/draggable/connected-draggable.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import { type Position } from 'css-box-model';
-import { type Node } from 'react';
+// eslint-disable-next-line
+import * as React from 'react';
```

Comment:
Is this needed? I have had issues with this in the past. Also, I have found just importing the type to be very clear

Reply:
Which issues? You use named imports all over the code. The dangerous may be using mixed imports like `import React, { Component } from 'react';` which is used all over the project too.